service/migration_manager.hh

```diff
@@ -80,6 +80,8 @@ public:
 
     migration_notifier& get_notifier() { return _notifier; }
     const migration_notifier& get_notifier() const { return _notifier; }
+    service::storage_proxy& get_storage_proxy() { return _storage_proxy; }
```

Comment:
I just wonder is adding a getter good candidate for separate commit? There is no regression risk, it won't make sense to backport, can't be seen how it will be used in the future. Perhaps there is some reason I am not aware of.

Reply:
It's a separate commit because I think it deserves a separate review - the commit message explains where  I plan to use it, but I want somebody who is more active in the "get rid of globals" effort (perhaps @avikivity) to see if this is perhaps a step in the wrong direction, or fine.

I thought that if I put these two lines in the middle of the larger patch that actually use this method, it will be easy to miss I modified migration_manager.hh.

Reply:
ok